src/obb/server/routes.clj

```diff
@@ -0,0 +1,13 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.routes
+  "Specifices the routes and associates handlers"
+  (:require
+    [compojure.core :as compojure :refer [GET]]
+    [compojure.route :as route]
+    [obb.server.handlers.index :as index]))
+
+(compojure/defroutes public-routes
+  "The routes available to be served, that don't need auth"
+  (GET "/" request (index/handler request))
```

Comment:
That's really interesting, never thought of that, but I also use `run` and not `runner` as a fn. I'll change that. It really reads better something like *index handle request*.